cli/src/main/kotlin/tools/samt/cli/DiagnosticFormatter.kt

```diff
@@ -115,20 +115,36 @@ internal class DiagnosticFormatter(
         DiagnosticSeverity.Info -> formatTextForSeverity("INFO:", severity, withBold = true)
     }
 
-    private fun formatGlobalMessage(message: DiagnosticGlobalMessage): String = buildString {
-        append(formatSeverityIndicator(message.severity))
+    private fun formatSeverityAndMessage(severity: DiagnosticSeverity, message: String): String = buildString {
+        // <severity>: <message>
+        append(formatSeverityIndicator(severity))
         append(" ")
-        append(message.message)
-        appendLine()
+
+        val indentLength = when (severity) {
+            DiagnosticSeverity.Error -> 7
+            DiagnosticSeverity.Warning -> 9
+            DiagnosticSeverity.Info -> 6
```

Comment:
These magic numbers relate to the length of their respective severity indicator right? (e.g. `7` == `"ERROR: ".length`

You could implement the indentLength as a size call instead if you want

Reply:
The problem with that is that the output of `formatSeverityIndicator` returns a string that is padded with the shell formatting tokens, resulting in an invalid length.

Reply:
I will add a comment to clarify what the magic numbers mean